pkg/collector/corechecks/snmp/internal/report/report_metrics.go

```diff
@@ -140,6 +140,10 @@ func (ms *MetricSender) reportColumnMetrics(metricConfig checkconfig.MetricsConf
 				tmpTags := common.CopyStrings(tags)
 				tmpTags = append(tmpTags, metricConfig.StaticTags...)
 				tmpTags = append(tmpTags, getTagsFromMetricTagConfigList(metricConfig.MetricTags, fullIndex, values)...)
+				if isInterfaceTableMetric(symbol.OID) {
+					interfaceCfg, _ := getInterfaceConfig(ms.interfaceConfigs, fullIndex, tmpTags)
```

Comment:
It would be nice to have a trace level log for the returned error ?

Reply:
Yes, I think a trace log makes sense